lib/linguist/languages.yml

```diff
@@ -1376,6 +1376,7 @@ GLSL:
   type: programming
   extensions:
   - ".glsl"
+  - ".comp"
```

Comment:
Are [these files](https://github.com/search?p=99&q=extension%3Acomp+NOT+void+NOT+main+NOT+version&type=Code&utf8=%E2%9C%93) GLSL files too?